subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -535,11 +746,25 @@ void ticker_worker(void *param)
 	/* Auto variable containing the head of tickers expiring */
 	ticker_id_head = instance->ticker_id_head;
 
+	/* Check if the previous ticker node which had air-time, is still
```

Comment:
Missing #if !defined(CONFIG_BT_TICKER_COMPATIBILITY_MODE)

Reply:
Will add the #if to avoid warnings. Is it possible for you to test scan_adv with nRF52 as well (=> new code exercised). As far as I can see, advertising is identical to legacy ticker.